src/sinks/clickhouse.rs

```diff
@@ -183,7 +184,7 @@ fn encode_uri(host: &str, database: &str, table: &str) -> crate::Result<Uri> {
     Ok(url.parse::<Uri>().context(super::UriParseError)?)
 }
 
-#[derive(Clone)]
+#[derive(Debug, Default, Clone)]
```

Comment:
Are these extra derivations (this and `HttpRetryLogic`) needed?

Reply:
Removed and implemented `Default` for `ClickhouseRetryLogic` outside `derive`.